package.json

```diff
@@ -10,8 +10,9 @@
 		"email": "sindresorhus@gmail.com",
 		"url": "https://sindresorhus.com"
 	},
+	"type": "module",
```

Comment:
I think it's a bit early to make this a `type` module. I prefer to start with less popular modules. Doing it here this early is just going to cause a lot of support requests.